sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1045,7 +1044,18 @@ static void _handle_nbr_sol(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
                     break;
             }
         }
-        reply_aro = _copy_and_handle_aro(netif, ipv6, nbr_sol, aro, sl2ao);
+        gnrc_pktsnip_t *reply_aro = NULL;
+        if (aro && sl2ao) {
+        /* If no SLLAO is included, then any included ARO is ignored. */
+            if (!(reply_aro = _copy_and_handle_aro(netif, ipv6, nbr_sol, aro, sl2ao))) {
+            /* If the Length field is not two, or if the Status field is not zero,
+               then the NS is silently ignored.*/
+                return;
+            }
```

Comment:
```suggestion
            reply_aro = _copy_and_handle_aro(netif, ipv6, nbr_sol, aro, sl2ao);
            /* If the Length field is not two, or if the Status field is not zero,
               then the NS is silently ignored.*/
```

Reply:
Is there a problem, why `_send_delayed_nbr_adv()` should not be called with (I guess you mean) `reply_aro == NULL`?

Reply:
Ah sorry I got confused and thought `gnrc_pktbuf_release(NULL)` was illegal when in fact that case is handled perfectly fine.

So everything should be good as is. Sorry for the noise.